lib/view_component/with_content_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  module WithContentHelper
+    def with_content(value = nil)
```

Comment:
Should we remove the default `nil` value if it's invalid?

Reply:
done!